tests/test_default_gaussian.py

```diff
@@ -323,7 +323,7 @@ def test_operation_map(self, gaussian_dev):
 
     def test_observable_map(self, gaussian_dev):
         """Test that default Gaussian device supports all PennyLane Gaussian continuous observables."""
-        assert set(qml.ops._cv__obs__)|{'Identity'}-{'Heterodyne'} == set(gaussian_dev._observable_map)
+        assert set(qml.ops._cv__obs__)-{'TensorN'}|{'Identity'}-{'Heterodyne'} == set(gaussian_dev._observable_map)
```

Comment:
😢 
Is it possible for `default.gaussian` to support this observable? This would allow it to be tested, and we are already doing a few non-Gaussian operations inside `default.gaussian` (such as fock probabilities).

Reply:
(@nquesada?)

Reply:
One could do it for two (and no more) number operators in different modes. Beyond that, it is a research question...

Reply:
Would be useful to add it for even two modes (and then you have to use SF if you want more modes).

Reply:
But this is likely a separate PR.

Reply:
This would require extracting logic that is now in SF and The Walrus. The original idea was to allow this operator to be used with PL-SF and to be able to include it in circuits when PennyLane is installed.

Reply:
Let's leave this for a future PR